src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -147,20 +147,17 @@ internal static SecurityZone GetFileSecurityZone(string filePath)
         {
```

Comment:
SecurityZone is a windows concept which I think shouldn't be applied when running on Unix plats. We shouldn't have `#if UNIX/endif` in this method or the methods exclusively used by this method, instead, we probably should analyze the call sites of this method to make sure this method doesn't run on unix plats, or better, enclose this method within `#if !UNIX/endif`.